spring-cql/src/test/java/org/springframework/cassandra/test/integration/core/cql/generator/TableLifecycleIntegrationTest.java

```diff
@@ -15,47 +15,39 @@
  */
 package org.springframework.cassandra.test.integration.core.cql.generator;
 
-import static org.springframework.cassandra.test.integration.core.cql.generator.CqlTableSpecificationAssertions.assertNoTable;
-import static org.springframework.cassandra.test.integration.core.cql.generator.CqlTableSpecificationAssertions.assertTable;
+import static org.springframework.cassandra.test.integration.core.cql.generator.CqlTableSpecificationAssertions.*;
 
-import org.cassandraunit.CassandraCQLUnit;
-import org.cassandraunit.dataset.cql.ClassPathCQLDataSet;
-import org.junit.Rule;
+import org.junit.Before;
 import org.junit.Test;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.cassandra.core.cql.generator.DropTableCqlGenerator;
 import org.springframework.cassandra.core.keyspace.DropTableSpecification;
 import org.springframework.cassandra.test.integration.AbstractKeyspaceCreatingIntegrationTest;
+import org.springframework.cassandra.test.integration.CqlDataSet;
 import org.springframework.cassandra.test.unit.core.cql.generator.AlterTableCqlGeneratorTests;
 import org.springframework.cassandra.test.unit.core.cql.generator.CreateTableCqlGeneratorTests;
 import org.springframework.cassandra.test.unit.core.cql.generator.DropTableCqlGeneratorTests;
 
 /**
  * Test CREATE TABLE / ALTER TABLE / DROP TABLE
- * 
+ *
  * @author David Webb
  * @author Oliver Gierke
+ * @author Mark Paluch
  */
 public class TableLifecycleIntegrationTest extends AbstractKeyspaceCreatingIntegrationTest {
 
 	private final static Logger log = LoggerFactory.getLogger(TableLifecycleIntegrationTest.class);
 
 	CreateTableCqlGeneratorTests.MultipleOptionsTest createTableTest = new CreateTableCqlGeneratorTests.MultipleOptionsTest();
 
-	public TableLifecycleIntegrationTest() {
-		super("tlit");
+	@Before
+	public void setUp() throws Exception {
+		cassandraRule
```

Comment:
A rather awkward line split (orphaned `execute` method), perhaps...

```
@Before
public void setUp() throws Exception {
  execute("cassandraOperationsTest-cql-dataload.cql", this.keyspace);
}

protected void execute(String cqlResourceName, String keyspace) {
  return execute(CqlDataSet.fromClassPath(cqlResourceName).executeIn(keyspsace));
}

protected void execute(CqlDataSet cqlDataSet) {
  cassandraRule.execute(cqlDataSet);
}
```

Then the `execute(cqlResourceName:String, keyspace:String)` and `execute(:CqlDataSet)` methods could be pulled up into the `AbstractKeyspaceCreatingIntegrationTest` and `AbstractEmbeddedCassandraIntegrationTest` classes as utility/helper methods, respectively.